website/content/devlog.smd

```diff
@@ -20,6 +20,16 @@ You can [subscribe to this page via RSS]($link.alternative('rss')).
 
 ## Changes
 
+## [Printer]($section.id('2025-07-07'))
```

Comment:
Why did you lower the heading level of all these? Shouldn't these be sub-headers of `Changes`?